.circleci/config.yml

```diff
@@ -432,15 +436,18 @@ workflows:
           requires:
             - deploy_to_doks
 
-  hourly_pipeline:
+  staging_pipeline:
```

Comment:
put this in a separate pr

Reply:
I have reverted this stuff and will return to it after we have the codacy-website publishing to dev sh working fine. 
It makes more sense to talk about the staging environment then.